.dockerignore

```diff
@@ -0,0 +1,22 @@
+**/__pycache__
+**/test_*
```

Comment:
One of the opinionated approaches I prefer is the pessimistic approach. Disallow all, then allow only the folders and files you require e.g

```
# ignore everthing
**
# allow the following
!apps
!requirements
!manage.py
# we finally again ignore the following from the folders allowed above
**/*__pycache__
```

Reply:
LGTM, I'll change it to much this

Reply:
I forgot to ignore tests folders again. You can confirm if we have missed any other important file or folder at the top level

```
# ignore everthing
**
# allow the following
!onadata
!requirements
!manage.py
!run.sh
# we finally again ignore the following from the folders allowed above
**/*test
**/*__pycache__
```